app/presenters/organisations/show_presenter.rb

```diff
@@ -0,0 +1,59 @@
+module Organisations
+  class ShowPresenter
+    attr_reader :org
```

Comment:
Yes, I think so. It gets called in prefixed_title - `prefix = needs_definite_article?(org.title) ? "the " : ""`

Reply:
You should be able to replace usages of org with @org

Reply:
I know, but there's loads of them in the code and it makes it slightly cleaner.

Reply:
It's a bit misleading though because it creates a getter method which could be called outside the class, which we don't need. Someone unfamiliar with the code may think this is an indication of it being needed elsewhere.

You could make it private, e.g:

```
private
    attr_reader :org

public
    #rest of presenter code goes here


```